modules/op-chains-generating-x509-secret.adoc

```diff
@@ -0,0 +1,55 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-generating-x509-secret_{context}"]
+= Generating the x509 key pair using TektonConfig CR
+
+To use the `x509` signing scheme for {tekton-chains} secrets, you must generate the `x509` key pair.
```

Comment:
@PuneetPunamiya can you please just check this one line that I just added based on @mramendi 's feedback? Thank you!

Reply:
LGTM